lib/serializer.js

```diff
@@ -144,21 +152,25 @@ function getSerializedContext(err) {
         return out.slice(0, -2);
     }
 
-    var ret = ' (';
+    var ret = '';
 
-    // construct context info
-    if (hasContext === true) {
-        ret += serializeIntoEqualString(err.context);
-    }
+    // look for error context in 3 places, in ascending order of precedence:
+    // 1) raw fields on the error object that are not known verror or
+    // restify-error fields
+    // 2) restify-error context fields (restify-errors@ <= 5.x)
+    // 3) verror info field
+    var rawFields = _.pickBy(err, function(val, key) {
+        return !_.includes(KNOWN_FIELDS, key);
+    });
```

Comment:
isn't it the same than: `var rawFields = _.omit(err, KNOWN_FIELDS);`

Reply:
oops good call